dmutils/content_loader.py

```diff
@@ -459,6 +459,9 @@ def _optional_form_fields(self):
         else:
             return []
 
+    def has_assurance(self):
+        return self.get('assuranceApproach', False)
```

Comment:
Yeah, I suppose.  I just cut and paste the existing logic, but you're probably right.